WBooks/Screens/BookLibrary/BookLibraryController.swift

```diff
@@ -34,6 +34,14 @@ class BookLibraryController: UIViewController {
         super.viewDidLoad()
         configureTableView()
         libraryItems = BookInfo.sharedInstance.getAllBooks()
+        
+        title = "LIBRARY".localized()
+        
+        let search = UIBarButtonItem(image: UIImage(named: "SearchIcon"), style: UIBarButtonItem.Style.plain, target: self, action: nil)
```

Comment:
You can create an `UIBarButtonItem` extension for these buttons. so you can reuse them